src/config.py

```diff
@@ -18,7 +18,6 @@
 
 DEFAULT_CONFIG_OPTIONS = """
 log.dirs=/var/snap/kafka/common/log
```

Comment:
I think `listener.name.sasl_plaintext.sasl.enabled.mechanisms=SCRAM-SHA-512` can be completely removed, both for TLS and non-TLS